PopPUNK/utils.py

```diff
@@ -161,7 +161,7 @@ def iterDistRows(refSeqs, querySeqs, self=True):
             for ref in refSeqs:
                 yield(ref, query)
 
-def listDistInts(refSeqs, querySeqs, self=True):
+def old_listDistInts(refSeqs, querySeqs, self=True):
```

Comment:
Can you just remove this function

Reply:
Sorry, just hadn't updated a local change, updated in 17a3f0e.